### PR TITLE
feat(react): #MAG-420 add base files

### PR DESCRIPTION
**Pull request by FlorentMr**
## Describe your changes
- update dependencies
- add const and enum
- add models
- init i18n process

Commented parts will be handled ulteriorly

## Checklist tests
do ```pnpm install``` command

## Issue ticket number and link
[MAG-420](https://jira.support-ent.fr/browse/MAG-420)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

**Review by alicedraillard — APPROVED** [review]

File: frontend/src/models/folder.model.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
+// import {model} from "entcore";
```
Comment (flmartineau) [inline frontend/src/models/folder.model.ts:1]: Retire le commentaire

File: frontend/src/models/folder.model.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    return this._shared;
+  }
+
+  set shared(value: any[]) {
+    this._shared = value;
+  }
+
+  isMyFolder(): boolean {
+    return false; //todo
+    // return this.ownerId === model.me.userId;
```
Comment (flmartineau) [inline frontend/src/models/folder.model.ts:84]: Remet cette ligne non ?

File: frontend/src/models/section.model.ts [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+  set cardIds(value: Array<string>) {
+    this._cardIds = value;
+  }
+
+  addCardId(value: string) {
+    this._cardIds.push(value);
+  }
+
+  // get displayed(): boolean {
```
Comment (flmartineau) [inline frontend/src/models/section.model.ts:99]: Normal que ça soit commenté ?

**Review by flmartineau — APPROVED** [review]

**Comment by flmartineau** [comment]
Pour les lignes commentées si c'est normal précise le peut être en commentaire

File: frontend/src/models/folder.model.ts (hunk 2 above)
Comment (alicedraillard) [reply frontend/src/models/folder.model.ts:84]: non car la partie pour récupérer les infos utilisateurs n'est pas encore mise en place

File: frontend/src/models/section.model.ts [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+    this._page = value;
+  }
+
+  get cards(): Card[] {
+    return this._cards;
+  }
+
+  // get displayed(): boolean {
+  //     return this._displayed;
+  // }
```
Comment (qperie) [inline frontend/src/models/section.model.ts:197]: Et lui, commenté pourquoi ?

**Review by qperie — APPROVED** [review]
JE rejoins Flavien, faudrait expliquer les lignes commentées

**Pull request merged**